edb/schema/annos.py

```diff
@@ -67,7 +67,7 @@ class AnnotationValue(
         so.Object, compcoef=1.0, default=None, inheritable=False)
 
     annotation = so.SchemaField(
-        Annotation, compcoef=0.429)
+        Annotation, compcoef=0.429, ddl_identity=True)
```

Comment:
Why is it called "DDL identity"? The name isn't obvious.

Reply:
The `Field` attribute has this comment:

```
    #: Whether the field is used to identify the object
    #: in DDL operations and schema reflection when object
    #: name is insufficient.
    ddl_identity: bool
```